lib/hooks/userspace/hook-express.js

```diff
@@ -106,28 +105,26 @@ function endRootSpanForRequest(rootContext, req, res) {
   rootContext.close();
 }
 
-module.exports = {
-  patch: function(express_, agent_, version_) {
-    if (!semver.satisfies(version_, SUPPORTED_VERSIONS)) {
-      agent_.logger.info('Express: unsupported version ' + version_ + ' loaded');
-      return;
-    }
-    if (!express) {
-      agent = agent_;
-      express = express_;
-      patchedMethods.forEach(function(method) {
-        shimmer.wrap(express.application, method, applicationActionWrap);
-      });
-    }
-  },
-  unpatch: function() {
-    if (express) {
-      patchedMethods.forEach(function(method) {
-        shimmer.unwrap(express.application, method);
-      });
-      agent.logger.info('Express: unpatched');
-      express = null;
-      agent = null;
-    }
+module.exports = function(version_, agent_) {
+  if (!semver.satisfies(version_, SUPPORTED_VERSIONS)) {
+    agent_.logger.info('Express: unsupported version ' + version_ + ' loaded');
+    return {};
   }
+  return {
+    'index.js': {
```

Comment:
This makes me uncomfortable. We are dependent on the internal file layout for the express module. If the `index.js` filename changes in a minor version for whatever reason, we will stop working.

I can understand the need for depending, somewhat, on the internal layout for mongodb because they don't expose the Pool object, but we shouldn't over-constrain all the other modules we are patching.